manifest: cleanup

DelightReza opened the pull request:
Signed-off-by: MD Raza <raza231198@gmail.com>

Commit:
manifest: cleanup

Signed-off-by: MD Raza <raza231198@gmail.com>

CaptainThrowback committed:
Use TeamWin repo for system_sepolicy

Change-Id: Ibd3865ca93988497dda851b0b280aa4826776a7f

CaptainThrowback committed:
Fix comment on system_sepolicy repo

CaptainThrowback committed:
Add @phhusson's lptools repo for lptools support

Change-Id: I6bf0d0bf0f48f902a2e17afdd777797f6ad14479

manjotsidhu reviewed (approved):
Looks good, we need to test android-11.0 builds once.

DelightReza commented:
> Looks good, we need to test android-11.0 builds once.

Wait wut 😂 let me try on ci, works fine in local build

imranpopz commented:
Build tested with this manifest ?

DelightReza commented:
> Build tested with this manifest ?

Yes

CaptainThrowback committed:
twrp-extras: add qcom common repos

Change-Id: I4bbb6e7584920267d12180f3aab11249a04057e8

Commit:
README: Replace `git://` with `https://`

- git:// has been deprecated by GitHub:
   *  fatal: remote error:
      The unauthenticated git protocol on port 9418 is no longer supported.
      Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
      manifests: sleeping 4.0 seconds before retrying

- Switch to https://, since people may get confused while copy-pasting commands.

CaptainThrowback committed:
Add python3 repo
